src/server/plugins/router.ts

```diff
@@ -1,7 +1,8 @@
 import { type Request, type ResponseToolkit } from '@hapi/hapi'
 
+import { getFormMetadata } from './engine/services/formsService.js'
```

Comment:
Lost the `~/src` for some reason and moved up

```suggestion
import { getFormMetadata } from '~/src/server/plugins/engine/services/formsService.js'
```

Reply:
I removed it and restored it, it'd be from the IDE when importing from the context menu. Fixed.